ablateLibrary/mesh/boxMesh.cpp

```diff
@@ -25,8 +31,8 @@ ablate::mesh::BoxMesh::BoxMesh(std::string name, std::map<std::string, std::stri
         }
     }
 
-    DMPlexCreateBoxMesh(comm, dimensions, simplex ? PETSC_TRUE : PETSC_FALSE, &faces[0], &lower[0], &upper[0], &boundaryTypes[0], PETSC_TRUE, &dm) >> checkError;
-    DMSetOptionsPrefix(dm, name.c_str()) >> checkError;
+    DMPlexCreateBoxMesh(PETSC_COMM_WORLD, dimensions, simplex ? PETSC_TRUE : PETSC_FALSE, &faces[0], &lower[0], &upper[0], &boundaryTypes[0], PETSC_TRUE, &dm) >> checkError;
```

Comment:
Controlling everything explicitly here is probably good. We should consider making another class that just does
```
DMCreate(comm, &dm);
DMSetType(dm, DMPLEX);
DMSetFromOptions(dm);
```
which would allow us to create lots of meshes from PETSc options alone.

Reply:
Thanks!  I created a new issue for this #72.